### PR TITLE
Cassandra API

### PR DESCRIPTION
**Pull request by azizbouland**
@CSellers720 Adds save and read API, and a formatting script.
Let me know if you got any questions,
Thanks!

### CODE REVIEW

File: database/cassandra.js [hunk 1]
```diff
@@ -47,9 +48,82 @@ client.connect()
...
+      membercontributions,memberhelpful,reviewdate,reviewtitle,reviewtext,reviewtriptype,\
+      reviewpictures,reviewratings) VALUES \
+      (${review.id},${review.hotelId},'${review.responderOrg}','${review.responderPicture}',\
+      '${review.responderClose}','${review.responderDate}','${review.responderName}',\
+      '${review.responderPosition}','${review.responderText}',${review.memberId},\
+      '${review.memberImg}','${review.memberUserName}','${review.memberLocation}',\
+      ${review.memberContributions},${review.memberHelpful},'${review.reviewDate}',\
+      '${review.reviewTitle}','${review.reviewText}','${review.reviewTripType}',\
+      '${review.reviewPictures}',${review.reviewRatings}
+      )`;
```
Comment (CSellers720) [inline database/cassandra.js:76]: You could automate it like this:
```
const columns = [all_column_names];
let query = 'INSERT INTO reviews.hotels1 (';
let values = 'VALUES (';
for(let column of columns) {
  query.concat(column + ',');
  values.concat(review[column] + ',');
}
query.pop();
values.pop();
query += ') ' + values + ')';
```

That way you'd only have to change the array to update the full query. Probably not worth re-working it at this point, just thought I'd mention it.

File: database/mongo.js [hunk 2]
```diff
@@ -41,19 +41,19 @@ var save = (review) => {
...
   })
 }
 
-var getHotelReviews = (hotelId) => {
+var read = (hotelId) => {
   return new Promise((resolve, reject) => {
     Review.find({"responderInfo.hotelId": hotelId}).exec((err, results) => {
       if (err) {
-        reject(err);
+        reject(err.stack);
```
Comment (CSellers720) [inline database/mongo.js:56]: I've never seen this before! Does it just return the full stack trace when the error is thrown?

File: database/utils/dataFormatter.js [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+      responderText: reviews[i].respondertext
+    }
+    formattedReview.reviewInfo = {
+      reviewDate: reviews[i].reviewdate,
+      reviewPictures: reviewpictures,
+      reviewRatings: ratingFormatter(reviews[i].reviewratings),
+      reviewText: reviews[i].reviewtext,
+      reviewTitle: reviews[i].reviewtitle,
+      reviewTripType: reviews[i].reviewtriptype
+    }
```
Comment (CSellers720) [inline database/utils/dataFormatter.js:45]: I feel like there's probably a way to use destructuring to shrink this down a bit, but if it works then don't mess with it. It's perfectly readable.

**Review by CSellers720 — COMMENTED** [review]
Did you mean to upload all of those images to git?

Looks like you're making good progress. Reach out if there's anything specific you'd like to discuss.

File: database/cassandra.js (hunk 1 above)
Comment (azizbouland) [reply database/cassandra.js:76]: I like that! I might incorporate something like it

File: database/mongo.js (hunk 2 above)
Comment (azizbouland) [reply database/mongo.js:56]: Yes, it gives a more detailed error by showing the stack trace.

**Pull request merged**